doc/api/deprecations.md

```diff
@@ -600,6 +600,16 @@ The DebugContext will be removed in V8 soon and will not be available in Node
 
 *Note*: DebugContext was an experimental API.
 
+<a id="DEP0070"></a>
```

Comment:
I was thinking about documentation noise, but I yield to @addaleax's opinion that doing it "the usual way" makes sense.